fuse_examples/classification/knight/baseline/clinical_processor.py

```diff
@@ -76,11 +76,14 @@ def __call__(self, sample_desc: Hashable):
         elif sample_data['smoking_history'] == 'current_smoker':
             sample_data['smoking_history'] = 2
 
-        if sample_data['last_preop_egfr']['value'] == '>=90':
-            sample_data['last_preop_egfr'] = 90
+        if sample_data['last_preop_egfr'] is None:
+            sample_data['last_preop_egfr'] = -1 
         else:
-            sample_data['last_preop_egfr'] = sample_data['last_preop_egfr']['value']
```

Comment:
Do you know what is the distribution of this value?
Not sure if -1 is a good idea here

Reply:
-1 is out of the normal range. do you propose to use the median or something like that instead?